### PR TITLE
fix: renBTC doesn't exist on L2

### PR DESCRIPTION
**Pull request by liamzebedee**
Epic: Synthetixio/issues#209

### CODE REVIEW

**Review by vbstreetz — APPROVED** [review]
awesome. looking great 👍🏽

**Comment by clementbalestrat** [comment]
@vbstreetz I'll let you merge this one if you think it resolves an issue. Otherwise you can close it.

**Comment by liamzebedee** [comment]
@vbstreetz @clementbalestrat woops, forgot the context for this change - the fix would allow us to run staking with a local OVM setup, as I'm working on in https://github.com/Synthetixio/issues/issues/209.

**Pull request closed without merging**
